assignment-client/src/avatars/MixerAvatar.h

```diff
@@ -21,73 +21,24 @@ class ResourceRequest;
 class MixerAvatar : public AvatarData {
     Q_OBJECT
 public:
-    MixerAvatar();
+    inline MixerAvatar() {}
```

Comment:
Why this change?

Reply:
The entire contents of the MixerAvatar() constructor has been removed (and the .cpp file can likely be deleted as it's pretty empty).  At this point it's an empty constructor.  Alternative suggestions?

Reply:
Mm, fair enough